### DISCUSSION TITLE
[tw.com] Demote QR to "more"

### DISCUSSION THREAD

**Discussion by twMat** [TiddlyWiki/TiddlyWiki5#8221] [General]
On tiddlywiki.com , I guess showing the QR button made sense when it was a new feature. It *is* really cool! But it does clutter up the tool buttons row; not only are several of the other "more" buttons are used much more, but IMO the QR button really belongs as a sub-feature of the "export" button.  No disrespect - QR is cool! But then, what is *not* cool in TW!? ;-)

<img width="201" alt="image" src="https://github.com/Jermolene/TiddlyWiki5/assets/4534056/f06b636a-d986-4be3-a402-7f0f8497e978">

Thanks everyone.

**Comment by pmario** [comment]
It also does not make much sense on mobile devices. 
If a QR code is shown, I do expect the possibility to share the URL with a social-app. But there is no such possibility.
I'm also in favour of removing it from the default toolbar, or make it drastically more useful. 

Especially the "Too big" message is annoying. IMO there should be no option to create a QR-code if it is too big to be shown. 

![image](https://github.com/Jermolene/TiddlyWiki5/assets/374655/b9569b54-40bb-4a4c-86ad-b0e48cbdb8fb)
